### ISSUE TITLE
Crash on iOS 11 because of adding subviews directly to UIVisualEffectView

### ISSUE THREAD

**Issue by AndreasVerhoeven** [idnow/de.idnow.ios#30]
`-[IDNInfoViewController viewDidLoad]` directly adds a couple of labels onto a `UIVisualEffectView`, which throws an `NSInternalException` since iOS11 (note that this would output a warning on iOS10) and thus crashes the whole app.

The proper fix for this is to add the labels to the `UIVisualEffectView.contentView` instead.

Related crash log:
```
Exception Type:  EXC_CRASH (SIGABRT)
Exception Codes: 0x0000000000000000, 0x0000000000000000
Exception Note:  EXC_CORPSE_NOTIFY
Triggered by Thread:  0

Application Specific Information:
abort() called

Filtered syslog:
None found

Last Exception Backtrace:
0   CoreFoundation                	0x184427d38 __exceptionPreprocess + 124
1   libobjc.A.dylib               	0x18393c528 objc_exception_throw + 55
2   CoreFoundation                	0x184427c0c +[NSException raise:format:arguments:] + 103
3   Foundation                    	0x184db6c24 -[NSAssertionHandler handleFailureInMethod:object:file:lineNumber:description:] + 111
4   UIKit                         	0x18dec37a8 -[UIVisualEffectView _addSubview:positioned:relativeTo:] + 391
5   IDNow                         	0x102c82d90 0x102c20000 + 404880
6   UIKit                         	0x18d836bfc -[UIViewController loadViewIfRequired] + 1039
7   UIKit                         	0x18d8367d4 -[UIViewController view] + 27
8   IDNow                         	0x102c68598 0x102c20000 + 296344
9   IDNow                         	0x102c67e28 0x102c20000 + 294440
10  IDNow                         	0x102c6701c 0x102c20000 + 290844
11  UIKit                         	0x18d836bfc -[UIViewController loadViewIfRequired] + 1039
12  UIKit                         	0x18d8367d4 -[UIViewController view] + 27
13  UIKit                         	0x18dbceef8 -[UINavigationController preferredContentSize] + 171
14  UIKit                         	0x18db80a4c -[UIPresentationController preferredContentSizeDidChangeForChildContentContainer:] + 67
15  UIKit                         	0x18db7cb8c __56-[UIPresentationController runTransitionForCurrentState]_block_invoke + 139
16  UIKit                         	0x18daa7680 _runAfterCACommitDeferredBlocks + 291
17  UIKit                         	0x18da9ab24 _cleanUpAfterCAFlushAndRunDeferredBlocks + 287
18  UIKit                         	0x18d829808 _afterCACommitHandler + 131
19  CoreFoundation                	0x1843cf8b8 __CFRUNLOOP_IS_CALLING_OUT_TO_AN_OBSERVER_CALLBACK_FUNCTION__ + 31
20  CoreFoundation                	0x1843cd270 __CFRunLoopDoObservers + 411
21  CoreFoundation                	0x1843cd82c __CFRunLoopRun + 1291
22  CoreFoundation                	0x1842ee2d8 CFRunLoopRunSpecific + 435
```

**Comment by hkretzschmar** [comment]
Hi Andreas,

we have fixed the bug and will release a new version 3.2.2 as soon as possilbe.

Kind regards
Holger Kretzschmar

**Comment by hkretzschmar** [comment]
We released a new version 3.2.2 yesterday which solves this issue.

Kind regards
Holger Kretzschmar
